helm_deploy/values-dev.yaml

```diff
@@ -9,7 +9,7 @@ generic-service:
 
   env:
     APPLICATIONINSIGHTS_CONFIGURATION_FILE: applicationinsights.dev.json
-    API_BASE_URL_OAUTH: https://sign-in-dev.hmpps.service.justice.gov.uk/auth
+    HMPPS_AUTH_BASE_URI: https://sign-in-dev.hmpps.service.justice.gov.uk/auth
```

Comment:
Changed these to match the new naming convention that we're proposing goes into the Kotlin template.